.goreleaser.yaml

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env bash
```

Comment:
I don't think you want this shebanged as bash?

Reply:
Sloppy copy/paste... sigh